WeakAuras/Prototypes.lua

```diff
@@ -9768,6 +9768,11 @@ Private.event_prototypes = {
         tinsert(events, "PLAYER_DIFFICULTY_CHANGED")
       end
 
+      if trigger.use_loot_specialization_id ~= nil then
+        tinsert(events, "PLAYER_LOOT_SPEC_UPDATED")
+        tinsert(events, "PLAYER_TALENT_UPDATE")
```

Comment:
I don't think PLAYER_TALENT_UPDATE is needed

Reply:
That should have been PLAYER_SPECIALIZATION_CHANGED